lib/cocoapods-amimono/integrator.rb

```diff
@@ -65,5 +69,11 @@ def create_or_get_amimono_phase(application_target:, phase_name:, script:)
         shell_build_phase.shell_script = script
       end
     end
+
+    def generate_filelist_script(aggregated_target:)
+      dependencies = aggregated_target.specs.map(&:name).reject { |dependency| dependency.include? '/'}
```

Comment:
Here we ignore subspecs, which are mapped on `aggregated_target.specs` but they don't have their own build directory.